component-server-parent/component-server/src/main/java/org/talend/sdk/component/server/front/ComponentResourceImpl.java

```diff
@@ -566,9 +566,15 @@ public Map<String, String> migrate(final String id, final int version, final Map
                         .status(Status.NOT_FOUND)
                         .entity(new ErrorPayload(COMPONENT_MISSING, "Didn't find component " + id))
                         .build()));
-        if (version > comp.getVersion() || version == comp.getVersion()) {
+        if (version > comp.getVersion()) {
```

Comment:
Why not >=?
If the config version == the version, we shouldn't migrate it, right?

Reply:
Why do we call migration if versions ==

Reply:
There's a comment on jira issue concerning studio's migration handling.
Migration fails in that case and the job cannot be opened. 
Same behavior observed in slack topic before the revert.

Reply:
Oh my...
Okay